cmd/k8s-operator/ingress-for-pg.go

```diff
@@ -314,21 +313,22 @@ func (r *HAIngressReconciler) maybeProvision(ctx context.Context, hostname strin
 	}
 
 	vipSvc := &tailscale.VIPService{
-		Name:    serviceName,
-		Tags:    tags,
-		Ports:   vipPorts,
-		Comment: svcComment,
```

Comment:
Can we set the comment field to something like 'resource owned by Tailscale Kubernetes Operator, do not modify'